front/src/components/HomePage/HomePage.js

```diff
@@ -0,0 +1,126 @@
+                            wyniki całościowe oraz ustawi ranking na ich
+                            podstawie.
+                        </p>
+                    </div>
+                    <div class="HomePage__card">
+                        <FontAwesomeIcon
+                            className="HomePage__card-icon"
+                            icon={faDownload}
+                        />
+                        <h3 className="HomePage__card-heading">
+                            Wpisywanie wyników przez zawodników
+                        </h3>
+                        <p className="HomePage__card-content">
+                            Możesz udostępnić możliwość wpisywania wyników do
+                            systemu zawodnikom, przez co nie będziesz musiał
+                            przejmować się zbieraniem metryczek podczas zawodów.
+                        </p>
+                    </div>
+                    <div class="HomePage__card">
+                        <FontAwesomeIcon
+                            className="HomePage__card-icon"
+                            icon={faHistory}
+                        />
+                        <h3 className="HomePage__card-heading">
+                            Wyniki dostępne dla użytkowników na żywo
+                        </h3>
+                        <p className="HomePage__card-content">
+                            Po wpisaniu każdej serii w naszym systemie, gdy
+                            klikniesz "podlicz wyniki", udostępniasz najnowszy
+                            ranking wszystkim osobą, które wejdą na tą stronę.
+                        </p>
+                    </div>
+                    <div class="HomePage__card">
+                        <FontAwesomeIcon
+                            className="HomePage__card-icon"
+                            icon={faListOl}
+                        />
+                        <h3 className="HomePage__card-heading">
+                            Automatyczne generowanie rozstawienia
+                        </h3>
+                        <p className="HomePage__card-content">
+                            Wystarczy, że wpiszesz zawodników do naszego systemu
+                            i przydzielisz im kategorie, a nasz system
+                            wygeneruje rozstawienie za ciebie.
+                        </p>
+                    </div>
+                    <div class="HomePage__card">
+                        <FontAwesomeIcon
+                            className="HomePage__card-icon"
+                            icon={faMagic}
+                        />
+                        <h3 className="HomePage__card-heading">
+                            Automatyczne przetwarzanie eliminacji
+                        </h3>
+                        <p className="HomePage__card-content">
+                            Na podstawie wyników kwalifikacji, nasz system
```

Comment:
Repetition, needless comma
```suggestion
                            Na podstawie wyników kwalifikacji system
```